src/client/doc.h

```diff
@@ -62,17 +62,26 @@ constexpr const char* IsInitV = R"mydelimiter(
         )mydelimiter";
 
 constexpr const char* IsInitVI = R"mydelimiter(
-        2. **__init__** (*vert_coordinates*, *face_indices*, *face_count*)
+        2. **__init__** (*vert_coordinates*, *face_indices*, *face_count*, *holes*)
 
         Constructs an InitialShape by accepting a list of direct vertex coordinates, a list of the vertex
-        indices for each faces and a list of the indices count per face. The vertex order is expected to
-        be counter-clockwise.
+        indices for each faces and a list of the indices count per face. The vertex order is expected to 
+        be counter-clockwise. The last parameter, *holes*, is optional and allows defining holes polygons in faces.
+        It is a list of lists, that assign hole-faces to faces. It follows this structure:
+
+		``[[face-with-holes1-idx, hole-face-idx1, hole-face-idx2,...], ..., [face-with-holesN-idx, hole-face-idx1, hole-face-idx2, ...]]``
```

Comment:
maybe make the structure example a bit more readable like this:
`[[index-of-face1-with-holes, index-of-hole1-in-face1, index-of-hole2-in-face1, ...], ..., [index-of-faceN-with-holes, index-of-hole1-in-faceN, index-of-hole2-in-faceN, ...]]`